apps/mobile/src/components/Feed/FeedFilter.tsx

```diff
@@ -6,7 +6,7 @@ import { GalleryTouchableOpacity } from '../GalleryTouchableOpacity';
 import { Pill } from '../Pill';
 import { Typography } from '../Typography';
 
-export type ActiveFeed = 'Worldwide' | 'Following' | 'Profile' | 'Curated';
+export type ActiveFeed = 'Worldwide' | 'Following' | 'Profile' | 'For You';
```

Comment:
went ahead and changed these, was a bit weird to keep tracking `Curated`

Reply:
ohh nice thank you